<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Line Chart </string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="paintValuesCB" >
       <property name="text" >
        <string>Display Data Values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="lineTypeLB" >
         <property name="text" >
          <string>Line Chart Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lineTypeCB" >
         <item>
          <property name="text" >
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Stacked</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Percent</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="paintLinesCB" >
         <property name="text" >
          <string>Paint Lines</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="markersLB" >
         <property name="text" >
          <string>Markers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="paintMarkersCB" >
         <property name="text" >
          <string>Paint Markers</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="markersStyleLB" >
         <property name="text" >
          <string>Markers Style</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="markersStyleCB" >
         <item>
          <property name="text" >
           <string>Map</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Circle</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Square</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Diamond</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>1 Pixel</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>4 Pixels</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Ring</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Cross</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Fast Cross</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="markersSizeLB" >
         <property name="text" >
          <string>Markers Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="markersWidthLB" >
             <property name="text" >
              <string>Width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="markersHeightLB" >
             <property name="text" >
              <string>Height</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QSpinBox" name="markersWidthSB" >
             <property name="value" >
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="markersHeightSB" >
             <property name="value" >
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>159</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="chartFrame" >
      <property name="minimumSize" >
       <size>
        <width>300</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
